cli/src/cmd_net.rs

```diff
@@ -420,6 +434,275 @@ impl AuthenticatedCmd for CmdBgpFilter {
     }
 }
 
+#[derive(Parser, Debug, Clone)]
+#[command(verbatim_doc_comment)]
+#[command(name = "auth")]
+pub struct CmdBgpAuth {
+    /// Id of the rack to add the auth config to.
+    #[arg(long)]
+    rack: Uuid,
+
+    /// Switch to add the auth config to.
+    #[arg(long, value_enum)]
+    switch: Switch,
+
+    /// Port to add the auth config to.
+    #[arg(long, value_enum)]
+    port: Port,
+
+    /// Peer to apply allow list to.
```

Comment:
See comment "allow list" for cli/docs/cli.json

Reply:
Fixed